build/sass.js

```diff
@@ -1,6 +1,9 @@
+const sass = require('node-sass');
+
 module.exports = function (grunt) {
     const options = {
         style: 'expanded',
+        implementation: sass,
```

Comment:
what does this do?

Reply:
the new grunt-sass needs implementation passed as an argument in the option, the sass implementation comes from node-sass dependency